### DISCUSSION TITLE
Disabling certain types of attachments in MessageInput widget

### DISCUSSION THREAD

**Discussion by bhayatus** [GetStream/stream-chat-flutter#759] [Ideas]
Hi, I'd like to request a feature if possible. We are currently using the out of box `MessageInput` widget for our active chat screen, and everything is working great. We are unable to tweak one thing we require though, the ability to only allow file + image attachments, and not allow video attachments. Would it be possible to add this feature?

**Comment by imtoori** [comment]
yes, we're working on this and will probably be there in the next 4.x version of the package

**Comment by grihlo** [comment]
Any update on that? Uploading and recording a video crashes the app for some iPhone users. We don't want to set `disableAttachments` to `true` since we want still users to be able to upload files and pictures.
